src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -73,13 +74,13 @@
 	private final JComboBox<String> toolType = new JComboBox<>(
 			new String[] { "Pickaxe", "Axe", "Sword", "Spade", "Hoe", "Shears", "Special", "MultiTool" });
 
-	private final JCheckBox stayInGridWhenCrafting = new JCheckBox("Check to enable");
-	private final JCheckBox damageOnCrafting = new JCheckBox("Check to enable");
+	private final JCheckBox stayInGridWhenCrafting = new JCheckBox(L10N.t("elementgui.common.check_to_enable_functionality"));
+	private final JCheckBox damageOnCrafting = new JCheckBox(L10N.t("elementgui.common.check_to_enable_functionality"));
 
-	private final Model normal = new Model.BuiltInModel("Normal");
+	private final Model normal = new Model.BuiltInModel(L10N.t("elementgui.tool.model_normal"));

```

Comment:
This should not be externalized as it is used in workspace save files

Reply:
Done!